doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -113,3 +113,9 @@ From NGSIv2 specification section "Special metadata in notifications", regarding
 
 Current Orion implementation supports "update" and "append". The "delete" case will be
 supported upon completion of [this issue](https://github.com/telefonicaid/fiware-orion/issues/1494).
+
+# URI parameters
+
+A special value of the URI param `options` (options=noAttrDetail) has been invented, for users of the
+broker to be able to avoid the CPU-intense query of the list of attribute types. `noAttrDetails` exists not
+for NGSIv2 but to improve performance in some operations.
```

Comment:
I'd suggest the following alternative:

```
# `noAttrDetail` option

The `noAttrDetail` option may be used in order to avoid NGSIv2 type browing queries (`GET /v2/types` and `GET /v2/types/<type>`) to provide attribute type detail. In that case, the `types` list associated to each attribute name is set to `[]`.

Using this option, Orion can solve these queries much faster, specially in the cases of a large number of attributes, each one with a different type. This can be very useful if your use case doesn't need the attribute type detail. We have tested in some cases savings from 30 seconds to 0.5 seconds using the `noAttrDetails` option.
```